src/test/mir-opt/enum_opt.rs

```diff
@@ -0,0 +1,78 @@
+// EMIT_MIR_FOR_EACH_BIT_WIDTH
+#![feature(arbitrary_enum_discriminant, repr128)]
```

Comment:
We do want to have some kind of test for this MIR-opt so you'll need to add

```suggestion
// EMIT_MIR_FOR_EACH_BIT_WIDTH
// compile-flags: -Zunsound-mir-opts
#![feature(arbitrary_enum_discriminant, repr128)]
```

so that the pass is enabled for this test.

Reply:
Oops yes that's my bad